app/controllers/api/v1/participants_controller.rb

```diff
@@ -74,7 +74,7 @@ def participant_action_params
           .require(:attributes)
           .permit(:course_identifier, :reason, :schedule_identifier, :cohort)
           .merge(
-            participant:,
+            participant_id: participant.id,
```

Comment:
Similar for the other controllers

Reply:
We would still have to find the participant again, unless I'm missing something? I opted to use the NPQ `id` rather than `ecf_id` here as I figure we should be consistent outside the controllers and not all models have an `ecf_id`

Reply:
You're right best not to rely on `ecf_id`